ussologin/visitwebpage.go

```diff
@@ -22,33 +20,53 @@ import (
 // Existing oauth tokens can be obtained, or new ones stored If non-nil, the
 // given TokenStore is used to store the oauth token obtained during the login
 // process so that less interaction may be required in future.
-func VisitWebPage(filler form.Filler, client *http.Client, store TokenStore) func(*url.URL) error {
+func VisitWebPage(tokenName string, client *http.Client, filler form.Filler, store TokenStore) func(*url.URL) error {
```

Comment:
@mhilton @rogpeppe this function probably doesn't live here, but it should live somewhere